### DISCUSSION TITLE
My Foundry Compiler is not Resolving Openzeppelin/contracts-Upgradeable

### DISCUSSION THREAD

**Discussion by tushar1698** [Cyfrin/foundry-full-course-cu#2257] [Q&A]
I have configured the OpenZeppelin Upgradeable contracts in one of my projects and done the remappings as well, but the compiler is still throwing errors that ERC20Upgradeable.sol: No such file or directory. I am dropping the code of my Remappings below. Please help. 
Note that I have installed both OpenZeppelin and OpenZeppelin Upgradeables. 
```````
[profile.default]
src = "src"
out = "out"
libs = ["lib"]

remappings = [
    "@openzeppelin/=lib/openzeppelin-contracts/contracts/",
    "@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts"
]

``````````

**Comment by EngrPips** [comment]
Hello @tushar1698, Please use `cmd + p` and enter `ERC20Upgradeable` in the search bar that appears. Like that you can use the file path to make the import. You can ignore remapping if it is causing you issues and import with the path of `ERC20Upgradeable`

**Reply by tushar1698** [reply]
Hey @EngrPips Thanks for the insight, It works but the problem is That I am auditing a Code, and I cannot change the imports, I am having the same issue with the normal OpenZeppelin contracts as well. I suspect that it has something to do with the version of the imports the client is using. This way I will need to change every import file manually. Is there any other way?

**Reply by EngrPips** [reply]
I am not sure what would be the alternative to changing the import of the file manually.
